markdown/1.x/en/documentation/currency-guide.md

```diff
@@ -103,6 +103,6 @@ Specify the maximum number of digits for all prices.
 ---
 #### **add-unit-to-filter-groups (Boolean) (deprecated since 1.3.1)**
 ~~If present, the unit formatting takes place for all filter groups and its filters.~~
-Since 1.2.2 the formatting takes places for all filter groups and its filters internally for every response. Setting the attribute was a noop or could lead to false encoding in case of the currency value differs on client and FACT-Finder site. Since 1.3.1 this attribute is always a noop.
+Since 1.2.2, for every response, the formatting is applied internally to all filter groups and their filters. Hence this attribute had no effect anymore or could lead to incorrect duplicative encoding. Since 1.3.1 this attribute has always no effect.
```

Comment:
It's the setting of the attribute that caused the undesired effects, isn't it?
There were two possible outcomes, so an `either...or` clause it adequate.
```
Hence setting this attribute either had no effect at all or it caused incorrect encoding.
```
"...no effect **at all**" emphasizes the no-op behaviour.
The fact that it was _duplicate_ encoding is irrelevant detail.

"always no effect......"

-> `this attribute...`
- `...has no effect`
- `...no longer has any effect`
- `...is ignored`

Reply:
> The fact that it was duplicate encoding is irrelevant detail.

is it? In case someone noticed the duplicated currency symbol it is easier to map the changelog entry for him with this information. Does it hurt to leave it there?

Reply:
When I read "duplicate encoding", I think of this `encodeURIComponent(encodeURIComponent(str))`. Nothing currency related.
When you say "encode", do you actually mean "adding a currency symbol"?

Reply:
I would personally map it. But obvious not clear, so let's skip the information

Reply:
done

feel free to _insert a suggestion_ directly next time. I totally love this new github feature , as it shortens feedback loop 👍